frontend/src/pages/workspace/document/Index.tsx

```diff
@@ -32,6 +37,18 @@ function DocumentIndex() {
 		};
 	}, [dispatch, client, doc]);
 
+	if (isLoading) {
+		return (
+			<Backdrop open>
+				<CircularProgress color="inherit" />
+			</Backdrop>
+		);
+	}
```

Comment:
We should wait both `useGetWorkspaceQuery` and `useGetDocumentQuery`.

Reply:
This too!